plugins/github-actions/src/components/WorkflowRunDetails/WorkflowRunDetails.tsx

```diff
@@ -116,8 +116,6 @@ const JobListItem = ({
     <Accordion TransitionProps={{ unmountOnExit: true }} className={className}>
       <AccordionSummary
         expandIcon={<ExpandMoreIcon />}
-        aria-controls={`panel-${name}-content`}
```

Comment:
Why are we removing these?

Reply:
Ah yeah good catch! See [this discussion](https://github.com/backstage/backstage/pull/14778#discussion_r1029333583), which I totally should have linked to in the PR to explain myself.